tests/wallet/cat_wallet/test_cat_outer_puzzle.py

```diff
@@ -62,5 +62,10 @@ def test_cat_outer_puzzle() -> None:
         ACS,
         inner_solution,
     )
-    double_cat_puzzle.run(solution)
+    try:
+        double_cat_puzzle.run(solution)
```

Comment:
Why should this be failing now?

Since I'm commenting here, this should use `with pytest.raises():` to require an exception get raised.  But, that will be taken care of in a followup as noted in the OP.

Reply:
We can actually remove this `double_cat_puzzle` stuff from the test entirely.  I was just using it as a way to test the recursiveness of the puzzle solving driver, but now that CATs don't work inside of each other, it's a less than ideal choice.

Reply:
Thanks for the info.  Are you saying that at this point this entire test is just verifying that we no longer have this feature?  Is it important to not have the feature?  Or did we just not make it work with CAT2.

Reply:
No the test is still validating that the cat driver works, I just wanted to check that it correctly passed some information to another layer, and since I already had the cat puzzle instantiated, I was using that.  The fix is to use a different puzzle's driver but for now it's okay to just delete all the double CAT stuff.  It was not important to have the CAT within a CAT feature.

Reply:
Actually after thinking on it 5 more seconds, leaving the raise catching is fine just with `pytest.raises(ValueError, match="clvm raise")` instead and that still tests what was intentioned